src/Tribe/Attendees_Table.php

```diff
@@ -380,13 +380,39 @@ public function column_check_in( $item ) {
 
 			return sprintf( $button_template, $item['order_id_link_src'], __( 'View order', 'event-tickets' ) );
 		}
+
+		$button_classes = ! empty( $item['order_status'] ) && in_array( $item['order_status'], $check_in_stati ) ?
+			'button-primary' : 'button-primary button-disabled';
+
 		if ( empty( $this->event ) ) {
-			$checkin   = sprintf( '<a href="#" data-attendee-id="%d" data-provider="%s" class="button-primary tickets_checkin">%s</a>', esc_attr( $item['attendee_id'] ), esc_attr( $item['provider'] ), esc_html__( 'Check in', 'event-tickets' ) );
-			$uncheckin = sprintf( '<span class="delete"><a href="#" data-attendee-id="%d" data-provider="%s" class="tickets_uncheckin">%s</a></span>', esc_attr( $item['attendee_id'] ), esc_attr( $item['provider'] ), esc_html__( 'Undo Check in', 'event-tickets' ) );
+			$checkin   = sprintf(
+				'<a href="#" data-attendee-id="%d" data-provider="%s" class="%s tickets_checkin">%s</a>',
+                esc_attr( $item['attendee_id'] ),
```

Comment:
Indentation is funky here (looks like maybe we've dropped into using spaces for the initial whitespace on this and the next few lines).